iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -783,6 +785,43 @@ public void testDropHiveTableWithoutUnderlyingTable() throws IOException {
     shell.executeStatement("DROP TABLE " + identifier);
   }
 
+  @Test
+  public void testAlterTableAddColumns() throws Exception {
+    Assume.assumeTrue("Iceberg - alter table/add column is only relevant for HiveCatalog",
+        testTableType == TestTables.TestTableType.HIVE_CATALOG);
+
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+
+    // Create HMS table with with a property to be translated
+    shell.executeStatement(String.format("CREATE EXTERNAL TABLE default.customers " +
+            "STORED BY ICEBERG " +
+            "TBLPROPERTIES ('%s'='%s', '%s'='%s', '%s'='%s')",
+        InputFormatConfig.TABLE_SCHEMA, SchemaParser.toJson(HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA),
+        InputFormatConfig.PARTITION_SPEC, PartitionSpecParser.toJson(SPEC),
+        InputFormatConfig.EXTERNAL_TABLE_PURGE, "false"));
+
+    shell.executeStatement("ALTER TABLE default.customers ADD COLUMNS " +
```

Comment:
Do we want to insert and read data after the alter in another test?

Reply:
There's a bit of an overlapping work on that here: https://jira.cloudera.com/browse/CDPD-25369, which should cover add/drop/rename column scenarios.

Reply:
Yeah I'd refrain from doing a thorough check of that here